flake.nix

```diff
@@ -3,14 +3,14 @@
 {
   description = "A collection of packages for the Nix package manager";
 
-  outputs = { self }:
+  inputs.lib = "path:./lib";
+
+  outputs = { self, lib }:
     let
       jobs = import ./pkgs/top-level/release.nix {
         nixpkgs = self;
       };
 
-      lib = import ./lib;
```

Comment:
We should keep this for compatibility (maybe as a forward, i.e. `lib = lib.lib`).

Reply:
done